src/lib/src/component/tree-viewer/tree-viewer.component.css

```diff
@@ -32,4 +32,16 @@
 
 .tree-view-item-key:hover {
     background-color: #2A2D2E;
+}
+
+.active {
+    background-color: #3f3f46 !important;
```

Comment:
Would check if this is really needed, its a hack, http://www.css-hack.de/CSS+allgemein/!+important:

"Bei großen Projekten oder für die schnelle Lösung kommen Sie manchmal nicht ohne ein ! important aus. Eleganter ist es aber immer, eine neue Klasse einzuführen, oder eine saubere Cascade zu schreiben."